modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/ApplicationTagListIntegrationTest.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright 2017 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.intuit.wasabi.tests.service;
+
+import com.intuit.wasabi.tests.library.TestBase;
+import com.intuit.wasabi.tests.library.util.serialstrategies.DefaultNameExclusionStrategy;
+import com.intuit.wasabi.tests.model.Experiment;
+import com.intuit.wasabi.tests.model.factory.ExperimentFactory;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+
+import static com.intuit.wasabi.tests.library.util.Constants.EXPERIMENT_STATE_DELETED;
+
+/**
+ * Integration Test for the retrieval of ExperimentTags by Application.
+ */
+public class ApplicationTagListIntegrationTest extends TestBase {
+
+    private Experiment experiment;
+    private String appName = "tagListTest-" + UUID.randomUUID();
+
+    /**
+     * Set up one experiment that has no tags.
+     */
+    @BeforeClass
+    public void setUp() {
```

Comment:
@AndreaSuckro I understand that this creates an experiment without tags? In my opinion, we should have tests for both scenarios i.e. Happy path (experiment with tags) and negative path (experiment without tags).